data/json/mapgen/lmoe.json

```diff
@@ -272,7 +272,6 @@
         { "point": "trap", "id": "tr_spike_pit", "x": [ 4, 19 ], "y": 7 },
         { "point": "trap", "id": "tr_spike_pit", "x": [ 3, 19 ], "y": 8 },
         { "point": "trap", "id": "tr_spike_pit", "x": [ 3, 20 ], "y": 9 },
-        { "point": "trap", "id": "tr_spike_pit", "x": [ 3, 20 ], "y": 10 },
```

Comment:
This line only belongs in your other PR?   Are you running multiple branches to prevent overlap?

Reply:
Oh... I think I may have made a branch of the branch instead of another branch off the master. Sorry git is new to me.

How should I fix this? Or just start over with a separate branch/pr for this one?

Reply:
I think I fixed it.